pkg/rancher-desktop/backend/containerClient/mobyClient.ts

```diff
@@ -74,10 +75,29 @@ export class MobyClient implements ContainerEngineClient {
     // fails to do so).
     while (successCount < 10) {
       try {
-        await this.runClient(['system', 'info'], 'ignore');
+        await this.runClient(['system', 'info'], 'pipe');
         successCount++;
+        failureCount = 0;
       } catch (ex) {
         successCount = 0;
+        failureCount++;
```

Comment:
I wonder if it would be interesting to log the `successCount` here, if it is greater than 0. Just to see if you get a couple of successes followed by a failure multiple times.

Reply:
`successCount` will always be 0 by the time we log, because we clear it on the first failure (and we only log after a few failures).  Because we call everything good after a stable run of successes (instead of flapping between success and failure).